includes/wcs-helper-functions.php

```diff
@@ -349,3 +349,15 @@ function wcs_compare_order_billing_shipping_address( $order ) {
 
 	return $addresses_are_equal;
 }
+
+/**
+ * Prefixes an order status with "wc-" if it is not already prefixed.
+ *
+ * @param string $order_status The order status
+ * @param string $prefix       The order status prefix. Optional. Default is "wc-".
+ *
+ * @return string The order status prefixed with "wc-".
+ */
+function wcs_prefix_order_status( $order_status, $prefix = 'wc-' ) {
```

Comment:
Even if the initial intended use case is dealing with order statuses, it feels like a pretty generic operation that could be applied in all sorts of use cases. Is there any merit in renaming to something like `wcs_prefix_string`?

Reply:
We have a `wcs_maybe_prefix_key()` function for the more generic case. I've updated this PR to just make use of it and add support for array of keys/strings.